sentinelone/plugin.spec.yaml

```diff
@@ -3,7 +3,7 @@ extension: plugin
 products: [insightconnect]
 name: sentinelone
 title: SentinelOne
-version: 2.1.1
+version: 2.1.2
```

Comment:
@r7-kszczepanskagorna input and output or action title changes require a major version bump